src/ElvSpace.js

```diff
@@ -59,60 +59,105 @@ class ElvSpace {
         console.log("create account - done");
       }
 
-      await elvAccount.Init({ privateKey: account.privateKey });
+      await elvAccount.Init({ privateKey: account.privateKey, tenantCheck: false });
 
       if (this.debug){
-        console.log("create admin group");
+        console.log("create admin groups");
       }
 
-      let adminGroup = await elvAccount.CreateAccessGroup({
+      // Create Tenant Admin Access Group
+      let tenantAdminGroup = await elvAccount.CreateAccessGroup({
         name: `${tenantName} Tenant Admins`,
       });
 
       await elvAccount.AddToAccessGroup({
-        groupAddress: adminGroup.address,
+        groupAddress: tenantAdminGroup.address,
         accountAddress: account.address,
         isManager: true,
       });
 
-      // Add KMS to tenant admins group
-      await elvAccount.AddToAccessGroup({
-        groupAddress: adminGroup.address,
-        accountAddress: this.kmsAddress,
+      await elvAccount.SetAccountTenantAdminsAddress({
+        tenantAdminsAddress: tenantAdminGroup.address,
       });
+      account.tenantAdminsId = await elvAccount.client.userProfileClient.TenantId();
 
-      await elvAccount.SetAccountTenantAdminsAddress({
-        tenantAdminsAddress: adminGroup.address,
+      // Create Content Admin Access Group
+      let contentAdminGroup = await elvAccount.CreateAccessGroup({
+        name: `${tenantName} Content Admins`,
       });
 
-      account.tenantAdminsId = await elvAccount.client.userProfileClient.TenantId();
+      await elvAccount.AddToAccessGroup({
+        groupAddress: contentAdminGroup.address,
+        accountAddress: account.address,
+        isManager: true,
+      });
 
       if (this.debug){
-        console.log("tenant admins:", adminGroup);
+        console.log("tenant admins:", tenantAdminGroup);
+        console.log("content admins:", contentAdminGroup);
       }
+      let adminGroups = {tenantAdminGroup, contentAdminGroup};
 
       let tenant = await this.TenantDeploy({
         tenantName,
         ownerAddress: account.address,
-        adminGroupAddress: adminGroup.address,
+        tenantAdminGroupAddress: tenantAdminGroup.address,
+        contentAdminGroupAddress: contentAdminGroup.address,
+      });
+
+      // We must sign in using the new account to set the groups' metadata - metadata can only be set by the groups' admin
+      let account_wallet = this.client.GenerateWallet();
+      let account_signer = account_wallet.AddAccount({
+        privateKey: account.privateKey,
+      });
+
+      this.client.SetSigner({ signer: account_signer, tenantCheck: false });
```

Comment:
Here we an use `elvAcount.client`
We already have a client object initialized with the new user account.  
So we can call:
elvAccount.client.SetAccountTenantContractID()
elvAccount.client.CallContractMethodAndWait() ...

Reply:
Thanks, I totally missed how elvAccount also has a client!